pkgs/ok_http/README.md

```diff
@@ -12,17 +12,23 @@ An Android Flutter plugin that provides access to the
 
 This size of the [example application][] APK file using different packages:
 
-| Package | APK Size (MiB) |
-|-|-|
-| **`ok_http`** | **20.3**  |
-| [`cronet_http`](https://pub.dev/packages/cronet_http) [^1] | 20.6 |
-| [`cronet_http` (embedded)](https://pub.dev/packages/cronet_http#use-embedded-cronet) [^2] | 34.4 |
-| `dart:io` [^3] | 20.4 |
+| Package                                                                                   | APK Size (MiB) |
+| ----------------------------------------------------------------------------------------- | -------------- |
+| **`ok_http`**                                                                             | **20.3**       |
+| [`cronet_http`](https://pub.dev/packages/cronet_http) [^1]                                | 20.6           |
+| [`cronet_http` (embedded)](https://pub.dev/packages/cronet_http#use-embedded-cronet) [^2] | 34.4           |
+| `dart:io` [^3]                                                                            | 20.4           |
 
 [^1]: Requires [Google Play Services][], which are not available on all devices.
 [^2]: Embeds the Cronet HTTP library.
 [^3]: Accessed through [`IOClient`](https://pub.dev/documentation/http/latest/io_client/IOClient-class.html).
 
+### 🔌 Supports WebSockets out of the box
+
+`package:ok_http` wraps the OkHttp [WebSocket][] API, allowing you to use it in your Flutter app with ease.
```

Comment:
Could you include what it will allow you to do in your Flutter app with ease?

Reply:
Done! Added a list of bullet points that showcases OkHttp features, and makes it easy to add more to the list. Left the table as it is (it looks cool, flexing over all other clients :) )